app/main.py

```diff
@@ -1 +1,29 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    """ That will move a file from one location to another """
+    # Ex: mv file.txt first_dir/second_dir/third_dir/file2.txt
+    if "mv" not in command:
+        print("Command mv not found")
+        return
+
+    dir_path = ""
+    file_in = command.split(" ")[1]
+    file_out = command.split(" ")[2]
+
+    if "/" in file_out:
+        for path in file_out.split("/")[:-1]:
+            dir_path += path + "/"
+            try:
+                os.mkdir(dir_path)
+            except OSError:
+                print("Error creating directory")
```

Comment:
try to use `os.makedirs` and `os.path.dirname`

Reply:
yes I tried but Im using windows and need to replace but if this program run on mac will be error.